src/hex_api_key.erl

```diff
@@ -13,13 +13,13 @@
 
 -type permission() :: api_permission() | repo_permission() | repos_permission().
 -ifdef(OTP_19).
--type api_permission() :: #{domain := api, resource => read | write}.
--type repo_permission() :: #{domain := repository, resource := binary()}.
--type repos_permission() :: #{domain := repositories}.
+-type api_permission() :: #{binary() := binary()}.
+-type repo_permission() :: #{binary() := binary()}.
+-type repos_permission() :: #{binary() := binary()}.
 -else.
--type api_permission() :: #{domain => api, resource => read | write}.
--type repo_permission() :: #{domain => repository, resource => binary()}.
--type repos_permission() :: #{domain => repositories}.
+-type api_permission() :: #{binary() => binary()}.
```

Comment:
We probably don't want these types anymore, but you tell me @wojtekmach, I would think you just want `permission()` now since it's all binary (keys and values).